var/spack/repos/builtin/packages/py-tables/package.py

```diff
@@ -25,15 +26,19 @@ class PyTables(PythonPackage):
     variant("lzo", default=False, description="Support for lzo compression")
 
     # requirements.txt
+    depends_on("python@3.6:", when="@3.7.0:", type=("build", "run"))
     depends_on("python@3.5:", when="@3.6.1:", type=("build", "run"))
     depends_on("python@3.4:", when="@3.6.0:", type=("build", "run"))
     depends_on("python@2.6:", type=("build", "run"))
 
     depends_on("py-setuptools", type=("build", "run"))
+    depends_on("py-setuptools@42.0:", when="@3.7.0:", type=("build", "run"))
     depends_on("py-cython@0.21:", type="build")
+    depends_on("py-cython@0.29.21:", when="@3.7.0:", type="build")
     depends_on("py-numpy@1.9.3:", type=("build", "run"))
     depends_on("py-numexpr@2.6.2:", type=("build", "run"))
     depends_on("py-six@1.9.0:", when="@:3.5", type=("build", "run"))
+    depends_on("py-packaging", when="@3.7.0:", type="build")
```

Comment:
```suggestion
    depends_on("py-packaging", when="@3.7.0:", type=("build", "run"))
```
Also included in `install_requires` and imported at run-time.